internal/indexer/postgres/affectedmanifest.go

```diff
@@ -120,6 +130,9 @@ func affectedManifests(ctx context.Context, pool *pgxpool.Pool, v claircore.Vuln
 		pkgsToFilter = append(pkgsToFilter, pkg)
 	}
 	log.Debug().Int("count", len(pkgsToFilter)).Msg("packages to filter")
+	if err := rows.Err(); err != nil {
```

Comment:
is this check redundant? you exhaust the rows in the loop above, is it still necessary for a final check of rows.Err here?

Reply:
I think so. This is how errors that occur while reading rows after a successful query are reported.

Reply:
wouldn't you encounter any possible errors on the Scan calls here: https://github.com/quay/claircore/pull/266/files/6c513e0421fcf451f93a880e8d91aef20d562804#diff-fe8b20110d1dedeb23c0974c2532aedf03f71871bfce78e7d485fcc8751ac417R109

Reply:
I'm pretty sure this is needed to catch an error where the connection drops after the Scan call. The Next call just returns a bool, and the Err call retrieves a stashed error.

Reply:
ahhh, I see.